atompub.go

```diff
@@ -37,17 +46,82 @@ type EventStoreContent struct {
 	Content     string    `xml:"content"`
 }
 
+//Are we configured to run in secure mode
+var kmsSvc *kms.KMS
+
+func CheckKMSConfig() error {
+	keyAlias := KeyAliasRoot + os.Getenv(KeyAlias)
+	if keyAlias == KeyAliasRoot {
+		return nil
+	}
+
+	_, err := encryptStuff(kmsSvc, []byte("test"), keyAlias)
+	if err != nil {
+		log.Warnf("Error in init test encrypt: %s - KMS config is unhealthy", err.Error())
+		return err
+	}
+
+	return nil
+}
+
+func init() {
+	keyAlias := KeyAliasRoot + os.Getenv(KeyAlias)
+
+	if keyAlias != "" {
```

Comment:
Fixed this and made the combo of key alias root and key alias uniform throughout.